tests/transformers/test_transformers_model_export.py

```diff
@@ -1341,6 +1366,81 @@ def test_qa_pipeline_pyfunc_load_and_infer(small_qa_pipeline, model_path, infere
     assert all(isinstance(element, str) for element in inference)
 
 
+def raw_image_file(imagename):
+    datasets_path = pathlib.Path(__file__).resolve().parent.parent.joinpath("datasets")
+
+    return datasets_path.joinpath(imagename).read_bytes()
+
+
+@pytest.mark.parametrize(
+    "inference_payload",
+    [
+        "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/cat.png",
+        [
+            "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/cat.png",
+            "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/cat.png",
+        ],
+        json.dumps(
+            {
+                "inputs": {
+                    "image": "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/cat.png",
+                },
+            }
+        ),
+        json.dumps(
+            {
+                "inputs": {
+                    "image": [
+                        "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/cat.png"
+                    ],
+                },
+            }
+        ),
+        raw_image_file("cat_image.jpg"),
+        [raw_image_file("cat_image.jpg"), raw_image_file("tiger_cat.jpg")],
+        json.dumps(
+            {
+                "inputs": {
+                    "image": base64.b64encode(raw_image_file("cat_image.jpg")).decode("ascii"),
+                },
+            }
+        ),
+        base64.b64encode(raw_image_file("cat_image.jpg")).decode("ascii"),
```

Comment:
Could you fix the failing tests? This becomes too long, we should convert it inside the test